common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -78,11 +78,8 @@
 %></%def>
 
 <%def name="include(path)"><%
-from django.conf import settings
-from django.template.engine import Engine
-from django.template.loaders.filesystem import Loader
-engine = Engine(dirs=settings.DEFAULT_TEMPLATE_ENGINE['DIRS'])
-source, template_path = Loader(engine).load_template_source(path)
+from django.template.loader import get_template
+source = get_template(path)
```

Comment:
@andy-armstrong This is the logic for how the underscore files are getting loaded by the mako templates.